src/Symfony/Component/Config/composer.json

```diff
@@ -19,6 +19,9 @@
         "php": ">=5.3.9",
         "symfony/filesystem": "~2.3"
     },
+    "require-dev": {
+        "symfony/yaml": "~2.7|~3.0"
```

Comment:
should be ~2.7 on 2.7,
~2.8|~3.0.0 on 2.8
and ~2.8|~3.0 on 3.1
should be done while merging up